package.json

```diff
@@ -12,7 +12,7 @@
     "test": "mocha test/ServerSpec.js & mocha test/setup.js test/ClientSpec.js",
     "dev": "webpack -d --watch",
     "build": "webpack -p",
-    "start": "node server/server.js",
+    "start": "nodemon server/server.js",
```

Comment:
Oh didn't even think how that would affect deployment... would it be helpful for me to change it back?